docs/source/serialization.mdx

```diff
@@ -400,10 +398,8 @@ to the library, you will need to:
 
 * Implement the ONNX configuration in the corresponding `configuration_<model_name>.py`
 file
-* Include the model architecture and corresponding features in
-[`onnx.features.FeatureManager`](https://github.com/huggingface/transformers/blob/c4fa908fa98c3d538462c537d29b7613dd71306e/src/transformers/onnx/features.py#L71)
-* Add your model architecture to the tests in
-`test_onnx_v2.py`
+* Include the model architecture and corresponding features in [`~onnx.features.FeatureManager`]
```

Comment:
This one is not documented so this link won't be resolved.

Reply:
Good catch! Will add the doc for this class too :)